### PR TITLE
added fluentd-elastisearch addon

### LINKED ISSUE #79432: Move fluentd-elasticsearch out of k/k
**What would you like to be added**:
currently the fluentd-elasticsearch cluster addon creates two docker images that are hosted in non-kubernetes controlled repositories and the code to create them is hosted in k/k.  either the code should be moved out of k/k or the images to should be controlled by the kubernetes org.

**Why is this needed**:
this type of ownership mismatch will lead to a bad user experience and possible security risks as and when the management for these artifacts get out of sync.

### PR DESCRIPTION
**Pull request by monotek**
Signed-off-by: André Bauer <monotek23@gmail.com>

* adds fluentd-elasticsearch from https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/fluentd-elasticsearch
* fluentd-elasticsearch will be remove in k/k in: https://github.com/kubernetes/kubernetes/pull/101039
* fixes: https://github.com/kubernetes/kubernetes/issues/79432

Todo / open for discussion if this should be part of this pr:
* add github action ci
* publish docker image to some registry controlled by k/k community members (currently quay.io is used)
* maybe add me and @coffeepac to the ownersfile / sig instrumentation so we can maintain the addon again?

### CODE REVIEW

**Review by logicalhan — APPROVED** [review]
/lgtm
/approve

**Review by ehashman — APPROVED** [review]
/lgtm

File: fluentd-elasticsearch/OWNERS [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- coffeepac
+- monotek
+- sig-instrumentation-approvers
+reviewers:
+- sig-instrumentation-reviewers
```
Comment (ehashman) [inline fluentd-elasticsearch/OWNERS]: You will need to define an OWNERS_ALIASES for these in the root of the repo for this to work.

See e.g. https://github.com/kubernetes-sigs/prometheus-adapter/blob/master/OWNERS_ALIASES

Comment (coffeepac) [reply fluentd-elasticsearch/OWNERS]: I think we'll need https://github.com/kubernetes-sigs/instrumentation-addons/pull/3 approved/merged first.  then we can run `/verify-owners` here and it'll get sorted.

Comment (monotek) [reply fluentd-elasticsearch/OWNERS]: If added the OWNERS_ALIAS file here too, as in https://github.com/kubernetes-sigs/instrumentation-addons/pull/2

If we only need it in the repo root, as suggested by @ehashman, i'll remove it again.

**Comment by monotek** [comment]
Note to myself: add changes from https://github.com/kubernetes/kubernetes/pull/100169

File: fluentd-elasticsearch/OWNERS_ALIAS [hunk 2]
```diff
@@ -0,0 +1,11 @@
+aliases:
```
Comment (ehashman) [inline fluentd-elasticsearch/OWNERS_ALIAS]: The file in the root needed to be OWNERS_ALIAS**ES** which is why it's not working. This file can be removed, the root file now is working.

**Comment by ehashman** [comment]
There we go. File wasn't named correctly. I fixed it in https://github.com/kubernetes-sigs/instrumentation-addons/commit/4726f83dc14cf1e386c7e7bccfd334b7abbe0b3f

**Review by ehashman — CHANGES_REQUESTED** [review]

**Comment by ehashman** [comment]
/hold

to remove the redundant OWNERS_ALIAS file

**Comment by monotek** [comment]
@ehashman 
OWNERS_ALIAS removed and updates from kubernetes/kubernetes#100169 added.

**Comment by monotek** [comment]
@ehashman 
ping :)

**Review by ehashman — COMMENTED** [review]
/hold cancel
/lgtm
/approve

**Pull request merged**